dashboard/app/models/gallery_activity.rb

```diff
@@ -25,6 +25,7 @@
 class GalleryActivity < ActiveRecord::Base
   belongs_to :user
   belongs_to :activity
+  belongs_to :level_source
```

Comment:
The "correct" analogy would be belonging to `:user_level` rather than `:level_source`, though it obviously breaks down because the `GalleryActivity` may belong to a previous activity rather than the current one.

Though I'm fine with (probably support) keeping as-is, the discussion should at least be had.